python/mage/max_flow/bfs_weight_min_max.py

```diff
@@ -0,0 +1,36 @@
+import mgp
+
+
+def BFS_find_weight_min_max(start_v: mgp.Vertex, edge_property: str) -> mgp.Number:
+    """
+    Breadth-first search for finding the largest and smallest edge weight,
+    largest being used for capacity scaling, and smallest for lower bound
+
+    :param start_v: starting vertex
+    :param edge_propery: str denoting the edge property used as weight
+
+    :return: Number, the largest value of edge_property in graph
+    """
+
+    next_queue = [start_v]
+    visited = set()
+    max_weight = 0
+    min_weight = float("Inf")
+
+    while next_queue:
+        current_v = next_queue.pop(0)
+        visited.add(current_v)
+
+        for e in current_v.out_edges:
+            # if there are edges without the given property, we ignore them,
+            # in order to support heterogeneous graphs
+            if edge_property not in e.properties:
```

Comment:
Don't forget to mention this in the documentation because traversing through the graph where we do not have a valid weight property would result in having `max_flow=0`